inc/class.client.php

```diff
@@ -63,6 +66,16 @@ public function __construct() {
 		return true;
 	}
 
+	function enqueue_frontend_scripts() {
+
+		wp_register_script('frontend.js', STAGS_URL . '/assets/js/frontend.js', array('jquery'), STAGS_VERSION);
```

Comment:
Just as I told you yesterday, do not use generic name as they'll conflict with other plugins. frontend.js and frontend.css are too generic as name to enqueue css and js.

Please always check existing code to see best practice especially when we've done similar thing before